doc/rtd/index.rst

```diff
@@ -95,14 +97,8 @@ projects, contributions, suggestions, fixes and constructive feedback.
    :hidden:
    :maxdepth: 1
 
-   Contributing overview <development/index.rst>
-   Contribute to code <development/contribute_code.rst>
-   Contribute to docs <development/contribute_docs.rst>
-   Contribute to testing <development/contribute_testing.rst>
-   Community <development/summit.rst>
-   Downstream packaging <development/packaging.rst>
```

Comment:
This is developer documentation. Hide these pages from user distraction under a deeper taxonomy that encourages developers to first read `index.html`.